ingestion-beam/bin/launch-22698806-extract

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -exo pipefail
+
+PROJECT="moz-fx-data-backfill-15"
+JOB_NAME="export-test2"
+
+mvn compile exec:java -Dexec.mainClass=com.mozilla.telemetry.SimpleSink -Dexec.args="\
+    --runner=Dataflow \
+    --jobName=$JOB_NAME \
+    --project=$PROJECT  \
+    --inputType=bigquery_table \
+    --input=moz-fx-data-shared-prod:payload_bytes_decoded.telemetry_telemetry__main_v4 \
+    --bqReadMethod=export \
```

Comment:
The three parameters above define the behavior of the input part of this pipeline. This script configures a job for reads via export to GCS.

All parameters below this line are irrelevant; they're there to configure the output of the Sink pipeline, but we've removed output for the purposes of this example.